lib/fortnox/api/models/invoice.rb

```diff
@@ -1,142 +1,43 @@
-require "fortnox/api/models/base"
+require "fortnox/api/models/document_base"
 require "fortnox/api/models/edi_information"
-require "fortnox/api/models/email_information"
-require "fortnox/api/models/row"
+require "fortnox/api/models/invoice_row"
```

Comment:
Question on this: is the invoice row and the other document rows different? If they are the same it should probably have some more generic name ...